### PR TITLE
Lesson 14

### PR DESCRIPTION
**Pull request by carlotrimarchi**

### CODE REVIEW

File: lesson14.md [hunk 1]
```diff
@@ -0,0 +1,511 @@
...
+
+### Recap: functions
+
+To use a function:
+
+1. first we need to create, define, a function
+  - function keyword
+  - function name
+  - parantheses ()
+  - optional parameters
```
Comment (haraldF) [inline lesson14.md]: no body?

File: lesson14.md [hunk 2]
```diff
@@ -0,0 +1,511 @@
...
+
+Well, not much. Nothing, actually.
+<!-- .element: class="fragment"  -->
+
+---
+
+### Static vs dynamic pages
+
+- HTML documents are static by nature
+- using CSS, we can change their appearance, make them, hopefully, prettier
```
Comment (haraldF) [inline lesson14.md]:
```suggestion
- using CSS, we can change their appearance and make them (hopefully) prettier
```

File: lesson14.md [hunk 3]
```diff
@@ -0,0 +1,511 @@
...
+
+1. getting access to the div element
+1. changing one of its properties, specifically the `backgroundColor`
+
+
+---
+
+### The DOM API in practice: getting access to an element
+
+Every operation in the DOM goes through the "document" object. And for now we
```
Comment (haraldF) [inline lesson14.md]:
```suggestion
Every operation in the DOM goes through the `document` object. And for now we
```

File: lesson14.md (hunk 1 above)
Comment (carlotrimarchi) [reply lesson14.md]: I figured that that's where most of the bugs usually are...

**Pull request merged**